src/main/java/eu/dissco/backend/service/ServiceUtils.java

```diff
@@ -0,0 +1,21 @@
+package eu.dissco.backend.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class ServiceUtils {
+  private ServiceUtils(){}
+
+  @Autowired
+  private static ObjectMapper mapper;
```

Comment:
Does this work? This is not really a utils functions as it depends upon another class (objectMapper). At some point it might be better to make a superclass for the services. If it works, let's leave it for now.

Reply:
Tested but it breaks for me.
If you want it to be a utils you would have to pass the mapper as an argument in the method.

Reply:
mapper added to function parameter